### PR TITLE
Spelling & article tweaks

### PR DESCRIPTION
**Pull request by emackey**
Most of these were found by Google Docs, and refined by me.

### CODE REVIEW

**Comment by emackey** [comment]
Except, of course, for the one found by Eric in https://github.com/KhronosGroup/glTF/issues/1668#issuecomment-933541686

File: specification/2.0/Specification.adoc [hunk 1]
```diff
@@ -818,13 +818,13 @@ Buffers and buffer views do not contain type information. They simply define the
 [[glb-stored-buffer]]
 ==== GLB-stored Buffer
 
-glTF asset **MAY** use GLB file container to pack glTF JSON and one glTF buffer into one file. Data for such buffer is provided via GLB-stored `BIN` chunk.
+The glTF asset **MAY** use a GLB file container to pack glTF JSON and one glTF buffer into one file. Data for such a buffer is provided via the GLB-stored `BIN` chunk.
```
Comment (lexaknyazev) [inline specification/2.0/Specification.adoc:821]: Are you sure about "use a GLB file container" here? A similar sentence on L474 says "use the GLB container".

Comment (emackey) [reply specification/2.0/Specification.adoc]: Agreed.  I changed it to "use the GLB container."

**Comment by emackey** [comment]
OK to merge?

**Pull request merged**